test/Driver/Http1DriverTest.php

```diff
@@ -251,8 +266,8 @@ public function testChunkedBodyParseEmit()
             $this->createCallback(0)
         );
 
-        for ($i = 0, $c = \strlen($msg); $i < $c; $i++) {
-            $promise = $parser->send($msg[$i]);
+        foreach (str_split($msg, $chunkSize) as $chunk) {
+            $promise = $parser->send($chunk);
         }
 
         while ($promise instanceof Promise) {
```

Comment:
This while should probably be inside the `foreach`, @trowski?

Reply:
Do you want me to change this @kelunik without @trowski's feedback? It looks like it'd make sense and the tests (still) pass with the `while`-loop moved inside.

Reply:
Yes, please.

Reply:
Done

Reply:
Yep, definitely belonged in the `foreach`.